ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/entity/QueryDescriptionFactoryTest.java

```diff
@@ -169,6 +170,7 @@ public void setUp() {
         0L,
         0L
     );
+    persistentQuery.initialize();
```

Comment:
For my understanding: why we need to call `initialize` now? It seems just building the topology / set exception handler?

Reply:
`queryStreams.localThreadsMetadata()` needed the KS object to be not null